config/samples/flows/jqtransformation/200-jqtransformation.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: flow.triggermesh.io/v1alpha1
+kind: JQTransformation
+metadata:
+  name: testjqtransform
+spec:
+  query: .foo | ..
```

Comment:
For my own curiosity, what does `| ..` do?

Reply:
`|` allows for chaining queries together and `..` will recursively descend into the object.  Given the example in the sample, `.foo` will retrieve the value keyed off `foo` while `..` will retrieve all objects. Granted it is a contrived example for this scenario as `.foo` would do the same thing.

Reply:
Exactly what I thought. The example contains unnecessary bits.

On the other hand, we could turn this example into an actual _transformation_. `jq` can do very powerful things.

I can come up with a query that transforms the JSON input into another JSON object if you like the idea.

Reply:
You're right it can, but at this point we just want to highlight that we can pass in a JQ style query. Did convert this into an iteration of arrays and remapping two fields if that helps.

Reply:
Nice 👍